.github/workflows/ci.yml

```diff
@@ -198,4 +280,4 @@ jobs:
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-kudu
       - name: Maven Tests
-        run: $RETRY ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-spark-launcher,presto-spark-package,presto-spark-testing -P test-presto-spark-integration-smoke-test
+        run: $RETRY ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-spark-launcher,presto-spark-package,presto-spark-testing -P test-presto-spark-integration-smoke-test
```

Comment:
new line is missing

Reply:
What does the presence or absence of a newline do?